components/content/LoadingText/LoadingText.stories.ts

```diff
@@ -0,0 +1,28 @@
+import { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import './LoadingText';
+
+export default {
+  component: 'diamond-loading-text',
+  argTypes: {
+    as: {
+      control: {
+        type: 'select',
+      },
+      options: ['h1', 'h2', 'h3', 'h4', 'p', 'span', 'div'],
+    },
+  },
+};
+
+export const Text: StoryObj = {
```

Comment:
Can you add some demos of different font size modifiers being applied.

Storyname should match the folder name I think